Form/Type/ModelAutocompleteType.php

```diff
@@ -132,6 +133,11 @@ public function configureOptions(OptionsResolver $resolver)
 
             'to_string_callback' => null,
 
+            // add button
+            // NEXT_MAJOR: Set this value to true to display button by default
+            'btn_add' => false,
```

Comment:
On other form types, the `btn_add` option is `false` when you want to disable the button, and any string when you want to enable it. This string is then the label of the button.

Please change the `NEXT_MAJOR` comment to `Set this value to 'link_add' to display button by default`

Reply:
Done